blog/2023-11-13-introducing-stately-sky/index.mdx

```diff
@@ -0,0 +1,72 @@
+---
+title: 'Introducing Stately Sky'
+description: Announcing Stately Sky, the new way to run your state machines in the cloud.
+authors: [gavin]
+tags: [new, sky, serverless, cloud, run, machines, state, stately]
+date: 2023-11-13
+slug: 2023-11-13-introducing-stately-sky
+image: /blog/2023-11-13-introducing-stately-sky.png
+---
+
+The Stately team is very excited to announce a new feature we’ve been working on for quite some time! Join us in welcoming Stately Sky to the Studio. Lovingly built with [PartyKit](https://www.partykit.io/), Sky is our new serverless platform for running workflows within the Studio. With Sky, users may now run their statecharts as live machines in minutes, complete with XState v5 actors and multiplayer support.
+
+<!--truncate-->
+
+## Why Sky: a seamless developer experience without the lock-in(Working title)
+
+### Developer Experience
+
+We know that state machines are extremely powerful for defining app logic, and Sky supercharges the Studio by allowing developers to **_build and deploy_** workflows to the edge, all defined as state machines! Typically, when we think of building workflows or long running processes, we typically think of 2 possible paths:
+
+- coding against large cloud services with cumbersome development environments, tons of boilerplate, and painful deployment processes.
+- relying on low-code or no-code platforms that tend to be inflexible and require vendor lock-in
+
+Sky aims to remove the barriers on both ends of the spectrum. We’ve always allowed developers to export their diagrams in the Studio as code, as JSON, and [more recently, as stories](https://stately.ai/docs/export-as-code#export-formats). There is no vendor lock-in from the Studio. Additionally, deploying workflows couldn’t be easier. Once the user has built and tested their machines in the Studio, they need only to generate an API key and click deploy.
+
+![GIF demonstrating flow generation process](./generateFlow.gif)
```

Comment:
When the new UI is in later this week, let’s redo this gif. And we can make it a bit narrower so the modal fills more of the image which will make it easier to read on mobile!

Reply:
Ah would be happy to. I resized it to be under 1K pixels wide, but happy to revise.